README.md

```diff
@@ -430,7 +430,31 @@ The hosted zone(s) should be configured with tags for cost tracking. A list of
 tags that should be provisioned is noted in
 [src/azul/deployment.py:tags](src/azul/deployment.py).
 
-### 3.1.3 Shared resources managed by Terraform
+### 3.1.3 AWS Chatbot integration with Slack
+
+AWS Chatbot integration with Slack enables monitoring of AWS Cloud events via
+AWS Chatbot app notifications. AWS Chatbot processes notifications from SNS
+topics, where alarms and cloud events are set up, and forwards them to the
+configured Slack channel. These notificatios in Slack are from `@aws`, the AWS
+Chatbot app. This serves the purpose of increasing the team's awareness of
+events and facilitating a rapid response to alarms. Enable this optional feature
+by populating the `azul_slack_integration` environment variable.
+
+Set up Chatbot through the AWS console for each Slack workspace in each AWS
+account. When configuring the Chatbot client, ensure that the Slack client type
+is selected. You'll be redirected to Slack's authorization page in order to
+request permission for AWS Chatbot to access the Slack workspace. Sign in with
+the account in the Slack workspace you want to integrate with. When that step is
```

Comment:
I don't understand what account this is referring to. The operator's slack account? a dedicated chatbot account?